src/python/tensorflow_cloud/__init__.py

```diff
@@ -19,3 +19,8 @@
 from .core.machine_config import MachineConfig
 from .core.run import run
 from .core.run import remote
+
+# Provides direct access to Tuner and CloudOracle classes,
+# which is the primary user-facing APIs.
+from .tuner.tuner import CloudOracle
+from .tuner.tuner import Tuner
```

Comment:
Should this be CloudTuner? Tuner seems very generic. CloudTuner is the implementation of KerasTuner that calls in CAIP optimizer right?

Reply:
The idea here was that since we are moving the code under `tensorflow_cloud `the calling it` tensorlfow_cloud.CloudTuner `would make it redundant,  assuming that the fact that this is a cloud based solution is implied by it being part of `tensorflow_cloud` package.

Reply:
Hmm, in that case should CloudOracle be just Oracle? One way or another we should make it consistent.

Reply:
good point. I will update it accordingly.